backend/.env.dist

```diff
@@ -61,7 +61,8 @@ EVENT_PROTOCOL_DISABLED=false
 # POSSIBLE VALUES: all | trace | debug | info | warn | error | fatal
 # LOG_LEVEL=info
 
-# DHT
-# if you set this value, the DHT hyperswarm will start to announce and listen
-# on an hash created from this tpoic
-# DHT_TOPIC=GRADIDO_HUB
+# Federation
+# if you set the value of FEDERATE_DHT_TOPIC, the DHT hyperswarm will start to announce and listen
+# on an hash created from this topic
+# FEDERATE_DHT_TOPIC=GRADIDO_HUB
+# FEDERATE_DHT_SEED=64ebcb0e3ad547848fef4197c6e2332f 
```

Comment:
I wondered in your PR already why you call those vars `FEDERATE_` instead of `FEDERATION_`. I think its valid to do so, but I expected the latter there - in the end chose as you please. I think its both valid.

Reply:
renamed to FEDERATION_xxx